lib/matplotlib/__init__.py

```diff
@@ -831,10 +831,16 @@ def matplotlib_fname():
         fname = os.path.join(configdir, 'matplotlibrc')
         if os.path.exists(fname):
             home = get_home()
+            depconfigdir = os.path.join(home, '.matplotlib')
+            link2xdg = os.path.islink(depconfigdir)
+            if link2xdg:
+                link2xdg = os.path.join(os.path.dirname(depconfigdir),
+                                        os.readlink(depconfigdir))
```

Comment:
And `os.path.realpath` on Windows does not follow symbolic links https://bugs.python.org/issue9949